charts/chart_versions.yaml

```diff
@@ -11,7 +11,7 @@ charts:
   - version: v3.29.101
     filename: /charts/rke2-calico-crd.yaml
     bootstrap: true
-  - version: 1.36.102
+  - version: 1.39.100
```

Comment:
No new images, only chart changes?

Reply:
yes, we had actually bumped the image before the upstream chart